core/src/main/java/io/kestra/core/storages/StorageInterface.java

```diff
@@ -67,7 +71,12 @@ default boolean exists(String tenantId, URI uri) {
     FileAttributes getAttributes(String tenantId, URI uri) throws IOException;
 
     @Retryable(includes = {IOException.class})
-    URI put(String tenantId, URI uri, InputStream data) throws IOException;
+    default URI put(String tenantId, URI uri, InputStream data) throws IOException {
```

Comment:
Would it be better to deprecate this one an always create the metadata? This can be handy for other functionalities.

Reply:
Idk since we never ever used metadata before maybe it's a better approach to leave it as-is and use the metadata one when there is a usecase for it ?